server/graphql/v2/mutation/CreateCollectiveMutations.js

```diff
@@ -51,6 +51,7 @@ async function createCollective(_, args, req) {
       if (!githubAccount) {
         throw new Error('You must have a connected GitHub Account to create a collective with GitHub.');
       }
+      // we skip actual Github verification for the fake test user e2e account since it is not possible to fake
```

Comment:
Without the context, this comment will be difficult to understand. Remove or rewrite. Ie: "NOTE: in e2e environment, checkGithubAdmin and checkGithubStars will be stubbed"